docs/configuration/data-validation.md

```diff
@@ -94,17 +94,17 @@ The following colors are supported:
 
 !!! tip "Dynamic Configuration Parameter"
 
-This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. An example is provided below:
+This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. The configuration can be carried out in the menu under “System/Configuration History/Custom validators”. 
+In the example, a device can only be deleted when it is set to the “decommissioning” status. Paths such as “dcim.device” can be found in the top right-hand corner when you open the page of a device. For a rack it would therefore be “dcim.rack”.
 
 ```python
-PROTECTION_RULES = {
-    "dcim.site": [
```

Comment:
Why change the object type?

Reply:
Yes ok you could leave it at Site. But I think the example is much more interesting for devices.
You decide if it bothers you, it can stay on site.

Reply:
I have an idea, I will reverse this example and integrate the variant for Device. Then you can see how to integrate multiple validations.
If that's OK?